Getting failures like

```
__________________ TestMethods.test_nargsort[last-expected0] ___________________
[gw1] linux -- Python 3.6.6 /venv/bin/python
self = <pandas.tests.extension.test_categorical.TestMethods object at 0xd6d702ec>
data_missing_for_sorting = [A, NaN, B]
Categories (2, object): [B < A]
na_position = 'last', expected = array([2, 0, 1], dtype=int64)
    @pytest.mark.parametrize('na_position, expected', [
        ('last', np.array([2, 0, 1], dtype='int64')),
        ('first', np.array([1, 2, 0], dtype='int64'))
    ])
    def test_nargsort(self, data_missing_for_sorting, na_position, expected):
        # GH 25439
        result = nargsort(data_missing_for_sorting, na_position=na_position)
>       tm.assert_numpy_array_equal(result, expected)
E       AssertionError: numpy array are different
E       
E       Attribute "dtype" are different
E       [left]:  int32
E       [right]: int64
```

--- a/pandas/tests/extension/base/methods.py
+++ b/pandas/tests/extension/base/methods.py
@@ -53,8 +53,8 @@ class BaseMethodsTests(BaseExtensionTests):
         self.assert_series_equal(result, expected)
 
     @pytest.mark.parametrize('na_position, expected', [
-        ('last', np.array([2, 0, 1], dtype='int')),
-        ('first', np.array([1, 2, 0], dtype='int'))
+        ('last', np.array([2, 0, 1], dtype=np.dtype('intp'))),
+        ('first', np.array([1, 2, 0], dtype=np.dtype('intp')))
     ])
     def test_nargsort(self, data_missing_for_sorting, na_position, expected):
         # GH 25439

